### ISSUE TITLE
Feature Request: Favorite Channel per game

### ISSUE THREAD

**Issue by byakuyaxgara** [DevilXD/TwitchDropsMiner#259]
I mainly use the miner for smite drops.

i would like to request the feature to favorite a channel so it earns points for it

So as there are many eligible channels for the drop. The miner will prioritize the favorite if live

**Comment by DevilXD** [comment]
Hello o/

This is a drops-focused application, without any particular "extra care" about channel points. Even claiming of the bonus points is a feature I added "just because" I felt that I can at least claim them if I'm going to receive them anyway. Earning drops can be done on any eligible channel though, and that isn't dependent on points nor any particular preference, other than maybe the amount of viewers a stream has - less viewed streamers tend to stream for shorter amounts of time, necessitating more unnecessary switches to be made.

That being said, it's not a definite "no" from me. I can consider adding it some time in the future.

**Comment by byakuyaxgara** [comment]
I do understand and agree with what u said.

It's just a bonus. You can take your time with it no problem if possible. Thanks for the effort

**Comment by notNSANE** [comment]
I wanted to suggest at least the option to let the app choose a random streamer instead of going to the top. Support snall streamers?

**Comment by DevilXD** [comment]
The current logic tries to minimize channel switches, by selecting a channel with the most viewers, as those channels tend to make long(er) streams, than people with 5-10 views that usually hop on for an hour or two, and then go offline. Again, the goal was to get drops in the most efficient way, regardless of what stream it's going to use for that purpose, and not doing unnecessary switches seemed the most logical.

Again, I'm not ruling out different logic as a possibility. For now, you can use the "Switch" button to manually select a different stream to watch, if you'd really like to focus on a particular channel. At least for a but, until the miner refreshes and switches it away anyway.

**Comment by notNSANE** [comment]
I agree with everything and it's the most logical way, of course. I just wanted to give a tiny boost to other streamers if possible. But there are other priorities.
